---
float64 distance_x
float64 distance_y
float64 avg_vel_x
float64 avg_vel_y
